### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by santinoncs**

### CODE REVIEW

**Comment by santinoncs** [comment]
build failed!!! I need to set up the environment variables...but travis is down...

File: templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        metadata:
+          labels:
+            app: cleanupdisk
+        spec:
+          serviceAccountName: {{ template "fullname" . }}
+          restartPolicy: Never
+          containers:
+          - image: softonic/kube-gcp-disks-roomba:{{ .Values.image.tag }}
+            args:
+	    {{- range .Values.args }}
```
Comment (rccrdpccl) [inline templates/cronjob.yaml]: Why is this red?

Comment (santinoncs) [reply templates/cronjob.yaml]: It had no default value.
Fixed in the following commit.

File: templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cleanupdisk
+  labels:
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  schedule: "0 10 * * *"
```
Comment (rccrdpccl) [inline templates/cronjob.yaml]: This should be parametrised

File: templates/clusterrole.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (rccrdpccl) [inline templates/clusterrole.yaml:1]: I guess we could make this more strict, but for now we can leave it

File: templates/cronjob.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: cleanupdisk
```
Comment (rccrdpccl) [inline templates/cronjob.yaml:4]: This probably should depend by release, although given the nature of this application is very unlikely that multiple releases are deployed in the same namespaces (or in the same cluster for that matter)

File: Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (rccrdpccl) [inline Chart.yaml]: Maybe better description :sweat_smile:

File: templates/cronjob.yaml (hunk 1 above)
Comment (santinoncs) [reply templates/cronjob.yaml]: This continues red

**Pull request merged**